packages/engine/paima-sm/src/index.ts

```diff
@@ -417,50 +417,50 @@ async function processUserInputs(
     // cancelDelegate = &wc|to_signature
     const delegateWallet = new DelegateWallet(DBConn);
     if (inputData.inputData.startsWith(DelegateWallet.INTERNAL_COMMAND_PREFIX)) {
-      await delegateWallet.process(
+      const status = await delegateWallet.process(
         inputData.realAddress,
         inputData.userAddress,
         inputData.inputData
       );
-    } else {
+      if (!status) continue;
+    } else if (inputData.userId === NO_USER_ID) {
       // If wallet does not exist in address table: create it.
-      if (inputData.userId === NO_USER_ID) {
-        const newAddress = await delegateWallet.createAddress(inputData.userAddress);
-        inputData.userId = newAddress.id;
+      const newAddress = await delegateWallet.createAddress(inputData.userAddress);
+      inputData.userId = newAddress.id;
+    }
+
+    // Trigger STF
+    const sqlQueries = await gameStateTransition(
+      inputData,
+      latestChainData.blockNumber,
+      randomnessGenerator,
+      DBConn
+    );
```

Comment:
What happens if the user's STF doesn't have something to parse this? What happens if the name conflicts with something else the user defined in their STF

Reply:
1. If the user does not have the parser setup, a console error will show up, but not halt paima-engine. (same error if a user posts any invalid command directly to the blockchain)

2. Conflict is unlikely as the command starts with the symbol `&` as described in the documentation.

<img width="531" alt="image" src="https://github.com/PaimaStudios/paima-engine/assets/3951895/5d8b49ed-9949-48e7-b61a-12116ff5ff8e">

Reply:
Okay, this may be a bit messy until we make STF failure more local instead of failing full txs, but we don't need to block this PR on that